src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -26,46 +26,46 @@
-    public static final String VALID_TYPE_AMY = "doctor";
-    public static final String VALID_TYPE_BOB = "patient";
-    public static final String VALID_TYPE_IRENE = "patient";
-    public static final String VALID_TYPE_JOHN = "donor";
-    public static final String VALID_AGE_IRENE = "21";
-    public static final String VALID_AGE_BOB = "21";
-    public static final String VALID_AGE_JOHN = "60";
-
-    public static final String NAME_DESC_AMY = " " + PREFIX_NAME + VALID_NAME_AMY;
-    public static final String NAME_DESC_BOB = " " + PREFIX_NAME + VALID_NAME_BOB;
-    public static final String PHONE_DESC_AMY = " " + PREFIX_PHONE + VALID_PHONE_AMY;
-    public static final String PHONE_DESC_BOB = " " + PREFIX_PHONE + VALID_PHONE_BOB;
-    public static final String NRIC_DESC_AMY = " " + PREFIX_NRIC + VALID_NRIC_AMY;
-    public static final String NRIC_DESC_BOB = " " + PREFIX_NRIC + VALID_NRIC_BOB;
-    public static final String TYPE_DESC_AMY = " " + PREFIX_TYPE + VALID_TYPE_AMY;
-    public static final String TYPE_DESC_BOB = " " + PREFIX_TYPE + VALID_TYPE_BOB;
-    public static final String AGE_DESC_BOB = " " + PREFIX_AGE + VALID_AGE_BOB;
-    public static final String NAME_DESC_IRENE = " " + PREFIX_NAME + VALID_NAME_IRENE;
-    public static final String PHONE_DESC_IRENE = " " + PREFIX_PHONE + VALID_PHONE_IRENE;
-    public static final String NRIC_DESC_IRENE = " " + PREFIX_NRIC + VALID_NRIC_IRENE;
-    public static final String TYPE_DESC_IRENE = " " + PREFIX_TYPE + VALID_TYPE_IRENE;
-    public static final String AGE_DESC_IRENE = " " + PREFIX_AGE + VALID_AGE_IRENE;
-    public static final String NAME_DESC_JOHN = " " + PREFIX_NAME + VALID_NAME_JOHN;
-    public static final String PHONE_DESC_JOHN = " " + PREFIX_PHONE + VALID_PHONE_JOHN;
-    public static final String NRIC_DESC_JOHN = " " + PREFIX_NRIC + VALID_NRIC_JOHN;
-    public static final String TYPE_DESC_JOHN = " " + PREFIX_TYPE + VALID_TYPE_JOHN;
-    public static final String AGE_DESC_JOHN = " " + PREFIX_AGE + VALID_AGE_JOHN;
+    public static final String VALID_NAME_PERSON_AMY = "Amy Bee";
+    public static final String VALID_NAME_PERSON_BOB = "Bob Choo";
+    public static final String VALID_NAME_PATIENT_IRENE = "Irene";
+    public static final String VALID_NAME_DONOR_JOHN = "John";
+    public static final String VALID_PHONE_PERSON_AMY = "11111111";
+    public static final String VALID_PHONE_PERSON_BOB = "22222222";
+
+    public static final String VALID_PHONE_PATIENT_IRENE = "85355255";
+    public static final String VALID_PHONE_DONOR_JOHN = "81230942";
+    public static final String VALID_NRIC_PERSON_AMY = "S1111111A";
+    public static final String VALID_NRIC_PERSON_BOB = "G2222222B";
+    public static final String VALID_NRIC_PATIENT_IRENE = "S1111111A";
+    public static final String VALID_NRIC_DONOR_JOHN = "T1312123P";
+    public static final String VALID_TYPE_PERSON_AMY = "doctor";
+    public static final String VALID_TYPE_PERSON_BOB = "patient";
```

Comment:
Yo Bob is a patient and amy is a doctor, can help change this?

Reply:
Change le